User can add ingredients to recipe

koggs opened the pull request:
### User adds ingredients
#### PT-Link: https://www.pivotaltracker.com/story/show/165101194

#### What I did:
- Create and implement feature
- Add foreign key migration
- Add edit view

koggs committed:
Add feature
Add ingerdient model

koggs committed:
update name of feature
implement basic steps

koggs committed:
updated views index, new and show

koggs committed:
Add recipe edit with a form
Add partial for ingredients

koggs committed:
Add foreign key migration
make ajax call to add ingredient to recipe

juliaramstedt reviewed (approved):
Looks very good! I think you might have some rubocop-errors, and maybe you need to add some config for cucumber to work in Travis?

juliaramstedt commented:
Unobtrusive JS is some pretty advanced stuff, so totally understand if you ran/run into issues with that!

koggs committed:
modifiy add ingredient metod

koggs committed:
Add gem cromedriver-helper and selenium-webdriver
Add cromedriver and selenium-webdriver configuration to features/support/env.rb

koggs committed:
modified add ingredient feature
modified model

koggs committed:
cange selenium configuration

koggs committed:
Add back link to views

koggs committed:
Add gem bootstrap and jquery-rails
add some styling to views
modified ajax call
